server/text_generation_server/layers/marlin/fp8.py

```diff
@@ -62,7 +62,7 @@ def from_unquant(cls, weight, bias, dtype):
         return cls(qweight=qweight, scales=scales.to(dtype), bias=bias)
 
     @classmethod
-    def from_fp8(cls, weight, scale, _input_scale, bias, dtype):
+    def from_fp8(cls, weight, scale, _input_scale, _scale_upper_bound, bias, dtype):
```

Comment:
Type.

These arguments get a bit messy. It's easy to mix up a tensor or a float (which was already happening here?). Maybe we should switch these to kwargs-only so that the call sites need to be explicit (+ type annotations).

Reply:
Converted them to kwargs and added type hints.